runs/models.tsp

```diff
@@ -143,9 +286,9 @@ model RunStepDetailsToolCallsObject {
 
   /**
    * An array of tool calls the run step was involved in. These can be associated with one of three
-   * types of tools: `code_interpreter`, `retrieval`, or `function`.
+   * types of tools: `code_interpreter`, `file_search`, or `function`.
    */
-  tool_calls: (RunStepDetailsToolCallsCodeObject | RunStepDetailsToolCallsRetrievalObject | RunStepDetailsToolCallsFunctionObject)[];
+  tool_calls: (RunStepDetailsToolCallsCodeObject | RunStepDetailsToolCallsFileSearchObject | RunStepDetailsToolCallsFunctionObject)[];
```

Comment:
```suggestion
  @extension("x-oaiExpandable", true)
  tool_calls: (RunStepDetailsToolCallsCodeObject | RunStepDetailsToolCallsFileSearchObject | RunStepDetailsToolCallsFunctionObject)[];
```

Reply:
Was missing the check on the array "items" element in the OpenAPI doc for the extension. Great catch!